followthemoney/util.py

```diff
@@ -1,7 +1,9 @@
+from __future__ import annotations
+
 import os
 import logging
 from threading import local
-from rdflib import Identifier  # type: ignore
+from rdflib.term import Identifier  # type: ignore
```

Comment:
Not sure what happened there, but it was failing for me. Did the library change or something?